server/tinode.conf

```diff
@@ -325,6 +325,9 @@
 				// Password to use when authenticating the sender; used only if "login" is provided.
 				"sender_password": "your-password-here",
 
+				// Mechanism supported for authentication we can use plain, login or md5
+				"mechanism": "login",
```

Comment:
Please rename to `auth_mechanism`, otherwise it is ambiguous without the comment.